js/sow.google-map.js

```diff
@@ -378,9 +380,14 @@ sowb.SiteOriginGoogleMap = function($) {
 						locationPromise.resolve( results[ 0 ].geometry.location );
 					} else if ( status === google.maps.GeocoderStatus.OVER_QUERY_LIMIT ) {
 						//try again please
-						if ( geocodetimer != null ) {
-							geocodetimer = setTimeout( function () {
-								this.getGeocoder().geocode.call( this, location, onGeocodeResults );
+						if ( geocodeTimer == null ) {
+							geocodeTimer = setInterval( function () {
```

Comment:
I attempted that initially but the problem is that it ran way too fast. I know it's not ideal, but it might be best if you attempt that as I feel I may have been overlooking something obvious.